frontend/src/app/modules/queue-dashboard/queue-dashboard.component.html

```diff
@@ -0,0 +1,34 @@
+<h1>
+  {{'frontend.de.iteratec.osm.queueDashboard.title' | translate}}
+</h1>
+<div class="card" *ngFor="let wptserver of wptServer$ | async" #thisCard isOpened="false" isLoaded="false">
+
+  <div>
+    <span>
+      <h2>{{wptserver.label}}</h2>
+      <span class="fas chevron-large"
+          (click)="thisCard.isOpened = !thisCard.isOpened"
+          [style.display]="thisCard.isLoaded ? 'block' : 'none'"
+          [class.fa-chevron-up]="thisCard.isOpened"
+          [class.fa-chevron-down]="!thisCard.isOpened"></span>
+    </span>
+  </div>
+
+  <div *ngIf="!thisCard.isLoaded">
+    <button id="load-data-button"
+            class="btn btn-primary pull-right"
+            (click)="loadQueueContent(wptserver.id);
+            thisCard.isOpened=true;
```

Comment:
It would be nicer if this logic would be inside the actual event handler `loadQueueContent`, so we minimize the code in templates. (You don't need to access `thisCard` if you use a simple member variable instead, see the comment above).